### ISSUE TITLE
[Enhancement] Add menu points to Edit for all entries of the editor's context menu [especially folding]

### ISSUE THREAD

**Issue by GitMensch** [OpenCobolIDE/OpenCobolIDE#178]
and for collapse all ([CTRL]+[SHIFT]+[-] works) / uncollapse all ([CTRL]+[SHIFT]+[+] works).

The menu points in Edit are mainly for the user to see the function and the shortcut.

**Comment by ColinDuquesnoy** [comment]
I am not sure I understand, do you mean collapse/uncollapse shortcut don't work (but collapse all/uncollapse all work)?

Those shortcuts are already in the editor context menu but I guess it would not hurt to add them into the edit menu as well.

**Comment by GitMensch** [comment]
I just tried some shortcuts and the (un)collapse all worked with the guessed shortcuts. The other shortcuts work fine, too.

I had only checked the menu for the functions and shortcuts :-)
I think it's a good design patter to have EVERY option in the menu and the options that may be useful in the current situation in the context menu (may be the same amount, may be less).

So this ticket is about placing every item of the context menu into the edit menu - should I rename the ticket?
